docsite/docs/origin_sdk/OriginSession.md

```diff
@@ -313,7 +313,7 @@ object).
 ```python
 @access_next_data_key_decorator
 def get_commodities(scenario_id: str,
-                    native_units_flag: bool = None,
+                    native_units_flag: Optional[bool] = None,
```

Comment:
This change was pending from the last update to the file. The argument is Optional in the function.